packages/phone-number-privacy/signer/src/salt-generation/get-salt.ts

```diff
@@ -9,6 +9,7 @@ import {
   phoneNumberHashIsValidIfExists,
 } from '../common/input-validation'
 import logger from '../common/logger'
+import { VERSION } from '../config'
```

Comment:
where is this defined?  I don't see VERSION added to the config.ts in the /signer directory

Reply:
It was added in a previous PR: https://github.com/celo-org/celo-monorepo/blob/aae2b147608ef1303df18b208c6778e01a22ccfd/packages/phone-number-privacy/signer/src/config.ts#L3